modules/apps/27-interchain-accounts/keeper/keeper_test.go

```diff
@@ -76,3 +76,24 @@ func (suite *KeeperTestSuite) TestGetPort() {
 	port := suite.chainA.GetSimApp().ICAKeeper.GetPort(suite.chainA.GetContext())
 	suite.Require().Equal(types.PortID, port)
 }
+
+func (suite *KeeperTestSuite) SetupICAPath(path *ibctesting.Path, owner string) error {
+	if err := InitInterchainAccount(path.EndpointA, owner); err != nil {
+		return err
+	}
+
+	if err := path.EndpointB.ChanOpenTry(); err != nil {
+		return err
+	}
+
+	if err := path.EndpointA.ChanOpenAck(); err != nil {
+		return err
+	}
+
+	if err := suite.chainB.GetSimApp().ICAKeeper.OnChanOpenConfirm(suite.chainA.GetContext(),
```

Comment:
`path.EndpointA.ChanOpenConfirm()` didn't seem to work here for me. I needed to use `simApp`